firebase.ts

```diff
@@ -0,0 +1,17 @@
+import firebase from "firebase/app";
+import "firebase/analytics";
+import "firebase/auth";
+
+const firebaseConfig = {
```

Comment:
This will have to be changed to the firebase config of the firebase app used for production.